icgcget/clients/commands/command_utils.py

```diff
@@ -0,0 +1,34 @@
+import click
+from ..errors import ApiError
```

Comment:
I don't understand. Why would that be the case? Is your problem that you are using relative imports?